PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+*Title* : *One line description*
+
+>Title can be a one or two words to describe the subsystem or the aspect
+ this PR applies to. For example: Build, Docs, HTTP, Upstream, SSl, ...
+
+
+*Description:* 
+>What does this PR do? What was the behavior before the PR?
+What is the behavior with the PR? If fixing a bug, please describe what
+the original issue is and how the change resolves it. How does this
+feature get enabled? By default, config change, etc...
+
+[Optional *Fixes* : # issue]
```

Comment:
I don't know if GH will auto close if there is a ':' between "fixes" and the issue. Not sure. You can test but then you might need to change this.

Reply:
Removed the `:` and the space after #.